cockatrice/src/handle_public_servers.cpp

```diff
@@ -0,0 +1,92 @@
+//
+// Created by Zachary Halpern on 4/6/18.
```

Comment:
git already knows this, comment isn't needed.

Reply:
CLion added it, didn't realize

Reply:
settings > new file template or so, to fix that for good